internal/build/lifecycle_execution.go

```diff
@@ -433,6 +444,8 @@ func (l *LifecycleExecution) Detect(ctx context.Context, phaseFactory PhaseFacto
 			CopyOutToMaybe(filepath.Join(l.mountPaths.layersDir(), "analyzed.toml"), l.tmpDir))),
 		If(l.hasExtensions(), WithPostContainerRunOperations(
 			CopyOutToMaybe(filepath.Join(l.mountPaths.layersDir(), "generated", "build"), l.tmpDir))),
+		If(l.hasExtensions(), WithPostContainerRunOperations(
+			CopyOutToMaybe(filepath.Join(l.mountPaths.layersDir(), "group.toml"), l.tmpDir))),
```

Comment:
I'm not able to leave a code suggestion, but lines 483-495 could become something like 

```
	kanikoCacheBindOp := NullOp()
	if l.platformAPI.AtLeast("0.12") {
		if l.runImageChanged() || l.hasExtensionsForRun() {
			registryImages = append(registryImages, l.runImageAfterExtensions())
		}
		if l.hasExtensionsForRun() {
			kanikoCacheBindOp = WithBinds(fmt.Sprintf("%s:%s", kanikoCache.Name(), l.mountPaths.kanikoCacheDir()))
		}
	}
```

We no longer need to provide `-build-image` to the restorer, because we don't need to fetch any information about it to provide to kaniko (since we're not using kaniko). This eliminates the requirement that the builder image must reside in a registry.

Reply:
To be fixed in https://github.com/itsdarshankumar/pack/pull/1

Reply:
done in latest commit

Reply:
done!